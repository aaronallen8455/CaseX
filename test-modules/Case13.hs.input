main :: IO ()
main = pure ()

test :: (Bool, Bool) -> Bool
test x = case x of
  SPLIT -> True
