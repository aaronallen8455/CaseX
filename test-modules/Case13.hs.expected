main :: IO ()
main = pure ()

test :: (Bool, Bool) -> Bool
test x = case x of
  (_, _) -> True
